pywbem/_cim_http.py

```diff
@@ -48,12 +49,19 @@
 from ._exceptions import ConnectionError, AuthError, TimeoutError, HTTPError, \
     HeaderParseError  # pylint: disable=redefined-builtin
 from ._utils import _ensure_unicode, _ensure_bytes, _format
+from ._warnings import RequestExceptionWarning
 
 __all__ = []
 
 _ON_RTD = os.environ.get('READTHEDOCS', None) == 'True'
 
-HTTP_CONNECT_TIMEOUT = 10        # HTTP connect timeout in seconds
+DEBUG_EXCEPTIONS = False  # Debug the behavior of requests/urllib3 exceptions
```

Comment:
Is it ever intended that a user use this to debug issues. If so may need more explanation.

Reply:
Not intended, it is only for our use. I added a statement clarifying that.